easybuild/easyconfigs/l/libStatGen/libStatGen-1.0.14-zlib.patch

```diff
@@ -0,0 +1,37 @@
+diff -ru libStatGen-1.0.14.orig/general/Makefile libStatGen-1.0.14/general/Makefile
```

Comment:
@pescobar please add a line of comment above this line, and add yourself as an author

```
use zlib installed with EasyBuild
author: Pablo Escobar (sciCORE, UniBas)
```

also, are you sure this is really needed? if the `zlib` module defines `$LIBRARY_PATH`, this should not be required?